packages/eas-cli/src/credentials/ios/api/graphql/mutations/AppleTeamMutation.ts

```diff
@@ -50,4 +52,40 @@ export const AppleTeamMutation = {
     );
```

Comment:
Can make same change to this function, and remove `CreateAppleTeamMutation` type (not needed, `account` not needed at any callsites)